lib/Importer/Caffe2ModelLoader.cpp

```diff
@@ -765,6 +765,37 @@ llvm::Error Caffe2ModelLoader::loadOperator(const caffe2::OperatorDef &op) {
     return llvm::Error::success();
   }
 
+  if (typeName == "ResizeNearest") {
+    NodeValue in;
+    ASSIGN_VALUE_OR_RETURN_ERR(in, getNodeValueByName(op.input(0)));
+
+    std::string order = "NCHW";
+    if (dict.count("order")) {
+      ASSIGN_VALUE_OR_RETURN_ERR(order, loadStr(dict["order"]));
+    }
+    // We expect the input to be NHWC.
+    NodeValue finalIn;
+    if (order == "NCHW") {
+      finalIn = G_.createTranspose(opName, in, NCHW2NHWC)->getResult();
+    } else {
+      finalIn = in;
+    }
+
+    RETURN_ERR_IF_NOT(dict.count("height_scale"),
```

Comment:
this isn't strictly necessary since loadFloat will return an error if it can't find height_scale or width_scale